@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -747,8 +748,8 @@ const onTemplateChange = () => {
 
   Object.assign(vmState, {
     isDiskTemplateSelected: isDiskTemplate,
-    vm_name: name_label,
-    vm_description: isDefaultTemplate ? '' : name_description,
+    name: name_label,
+    description: isDefaultTemplate ? '' : name_description,
```

Comment:
Why did you change these name properties?
This does not appear to be related to this PR